tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -109,6 +114,9 @@ def assert_job_succeeded(self, job_id, children_number=0):  # noqa: D102
         assert_that(status).is_length(1 + children_number)
         assert_that(status).contains_only("SUCCEEDED")
 
+    def compute_nodes_count(self):  # noqa: D102
```

Comment:
Do you have to implement this because it's in the interface?

Reply:
Yes and at the moment no test is using it.